companies/danskebank.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "danskebank",
+    "relevant-countries": [
+        "dk"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Danske Bank",
+    "address": "Danske Bank A/S, Holmens Kanal 2-12, 1092 København K, Denmark",
+    "phone": "+45 33 44 00 00",
+    "email": "danskebank@danskebank.dk",
```

Comment:
[This source](https://danskebank.dk/PDF/GDPR/Danske_Bank_privacy_notice.pdf) seems to have a better email address that would allow us to set the `suggested-transport-medium` to `email`.